client/driver/docker.go

```diff
@@ -138,11 +138,8 @@ type DockerLoggingOpts struct {
 type Mount struct {
```

Comment:
Oops, should have pointed out we should prefix these type names with `Docker...` since all of our drivers are in a single package. `:GoRename DockerMount` here and `:GoRename DockerBindOptions` below should do it with vim-go :)